app/src/main/java/cm/aptoide/pt/app/view/donations/view/DonateDialogFragment.java

```diff
@@ -91,9 +89,8 @@ public static DonateDialogFragment newInstance(String packageName, boolean hasWa
     ((MainActivity) getContext()).getActivityComponent()
         .inject(this);
     packageName = getArguments().getString(PACKAGE_NAME);
-    presenter = new DonateDialogPresenter(this, donationsService, walletService,
-        new CompositeSubscription(), AndroidSchedulers.mainThread(), appNavigator,
-        donationsAnalytics);
+    presenter = new DonateDialogPresenter(this, walletService, new CompositeSubscription(),
+        AndroidSchedulers.mainThread(), appNavigator, donationsAnalytics);
```

Comment:
This class is using SeekBar.getThumb() which requires API 16+. Should open a ticket for this to check behaviour on Ice Cream Sandwich devices and to decide to whether fix it or not since we are close to make the minimum api lvl 16.